src/MCMCMonitorDataManager/updateSequences.ts

```diff
@@ -1,25 +1,39 @@
-import { GetSequencesRequest, isGetSequencesResponse } from "../../service/src/types";
+import { GetSequencesRequest, isGetSequencesResponse, MCMCSequenceUpdate } from "../../service/src/types";
 import postApiRequest from "../networking/postApiRequest";
+import getSpaSequenceUpdates from "../spaInterface/getSpaSequenceUpdates";
 import { MCMCMonitorAction, MCMCMonitorData } from "./MCMCMonitorDataTypes";
 
-
 export default async function updateSequences(data: MCMCMonitorData, dispatch: (a: MCMCMonitorAction) => void) {
     const X = data.sequences.filter(s => (s.updateRequested))
     if (X.length > 0) {
-        const req: GetSequencesRequest = {
-            type: 'getSequencesRequest',
-            sequences: X.map(s => ({
-                runId: s.runId, chainId: s.chainId, variableName: s.variableName, position: s.data.length
-            }))
+        const runId = X[0].runId
```

Comment:
There's not actually anything enforcing that the `runId`s be homogeneous across all sequences--so technically it's insufficient to check the first one only (even though I have a hard time imagining that a situation where we have a mix would come about). However, I would recommend changing this to a `some`/`every` situation so that we can be explicit and catch it if the case does ever come up.